src/mappings/geyser.ts

```diff
@@ -12,7 +12,7 @@ import {
   GysrSpent
 } from '../../generated/templates/Geyser/Geyser'
```

Comment:
now that we are associating users with their owned geyser, it might also make sense to log geyser management events like adding funding as +1 to `User.operations`

Reply:
Good catch. I'll bump the `Geyser.operations` as well.